### DISCUSSION TITLE
Bypass replacing google play services

### DISCUSSION THREAD

**Discussion by sdre15** [lukaspieper/Gcam-Services-Provider#43] [General]
Is there anyway to install gcam services provider w/o completely removing google play service. I'm not familiar with Android development enviroment, but I think Google Cam calls `com.google.android.gms` to get the Google Play Services. And this service providers spoofs the identifier to provide Google Service to GCam. However, I still have play service installed and disabled, which prevents me from installing the application. Is there anyway I can bypass this. I've already changed the identifer to get the installation working and GCam still crashes which lead me to the conclusion above. Correct me if I'm wrong.

Perhaps there some kind of way to modify Google Camera Port to call a different identifer name?

**Comment by sdre15** [comment]
#19  I'll try the solution here. @lukaspieper can u build an artficat again or release it w/ the new changes

**Reply by lukaspieper** [reply]
Agree, I need to create a new release. Until then, GitHub already created a [new artifact 5 days ago](https://github.com/lukaspieper/Gcam-Services-Provider/actions/runs/1049170786).

However I am not sure if this will work. You cannot have multiple implementations of the same service installed at the same time. The package manager ("app installer") will prevent the installation if the service is already on the device available.
